### ISSUE TITLE
Auto attach Jolokia on application restart

### ISSUE THREAD

**Issue by Saurabh995** [rhuss/jolokia#412]
I have been trying to attach Jolokia dynamically using the following command
java -jar jolokia-jvm-1.3.4-agent.jar start <myapplication_regex>
This works successfully. But on application restart, Jolokia terminates.

Do we have any workaround to handle application restart case?

**Comment by grgrzybek** [comment]
There's no workaround - JVM Agent attachment is stateless - you simply choose a PID and attach the agent.
If the target application restarts, you need some kind of shell script that monitors the process and reattaches Jolokia JVM agent.
There's no plan to support this scenario in Jolokia itself.

**Comment by tadayosi** [comment]
I agree. I think this is something a launcher script or monitoring tool should take care, rather than in Jolokia.
